src/parse.js

```diff
@@ -120,8 +120,8 @@ export default function(urlStr, parseQs = false, slashesDenoteHost = false) {
   }
 
   res.slashes = slashes && !preSlash;
-  res.host = url.host.includes(HOST) ? '' : url.host;
-  res.hostname = url.hostname.includes(HOST)
+  res.host = ~url.host.indexOf(HOST) ? '' : url.host;
```

Comment:
I wonder if Microbundle should ship this transform? maybe under a loose flag?

https://astexplorer.net/#/gist/d0188a1c819b6e00e87f4faa15dd39c0/c4f870b043cadef9745b69724019727e5cf30669

Reply:
Only if they want IE11 support. Should be a babel thing IMO?